src/transform/activation.jl

```diff
@@ -0,0 +1,132 @@
+
+# -------------------------------------------------------------
+# Repurposed from https://github.com/tbreloff/OnlineAI.jl
+# -------------------------------------------------------------
+
+abstract Activation <: Transformation
+
+value(activation::Activation, input::AbstractVector) = Float64[value(activation, Σ) for Σ in input]
```

Comment:
I think an untyped comprehension will be consistently type stable in Julia 1.0... right now I'm not so sure.  The thing I want to avoid is a case when a vector of integers **doesn't** get auto-promoted to floating-point, as that's frequently what I would prefer.  I'll need to think through my argument a little better before I comment any further though... no time now.